### DISCUSSION TITLE
When will you support 8 digit notice numbers?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#315] [Q&A] [answered]
Hi,
We where informed on a workshop for TED data reusers 27/09/2022 that the publication number for eForms notices will have 8 digits (e.g. 00654321_2022.xml). When will you support the 8 digit numbers starting with 00?

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this non-technical question, I invite you to contact directly the Helpdesk: ted@publications.europa.eu
Thanks,
